easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-foss-2016a.eb

```diff
@@ -14,6 +14,10 @@ source_urls = [XORG_LIB_SOURCE]
 builddependencies = [
     ('kbproto', '1.0.7'),
     ('renderproto', '0.11'),
+    ('libX11', '1.6.3'),
```

Comment:
@SimonPinches the problem indicated by Jenkins is that you're including `libX11` both as build depenency and runtime dependency (it already was a runtime dep), so: just drop this line